mixer/adapter/opa/opa_integration_test.go

```diff
@@ -177,7 +177,8 @@ spec:
 `
 )
 
-func TestServer(t *testing.T) {
+//https://github.com/istio/istio/issues/2300
+func xTestServer(t *testing.T) {
```

Comment:
there is an outstanding PR to fix this. Should we merge that first and then drop this change?

Reply:
Either way - if it gets merged before I finish (likely), I'll drop.